tahoe_sites/tests/test_models.py

```diff
@@ -74,69 +79,31 @@ class TestTahoeSiteUUID(DefaultsForTestsMixin):
-        default_site = TahoeSiteUUID.objects.get(organization=self.default_org)
-        assert TahoeSiteUUID.get_organization_by_uuid(default_site.site_uuid) == self.default_org
-
-    @pytest.mark.skipif(settings.FEATURES['TAHOE_SITES_USE_ORGS_MODELS'],
-                        reason='Runs only when TAHOE_SITES_USE_ORGS_MODELS is off')
-    def test_get_uuid_by_organization_without_org(self):
-        """
-        Test get_uuid_by_organization helper when edx-organizations customization is off
-        """
-        default_site = TahoeSiteUUID.objects.get(organization=self.default_org)
-        assert TahoeSiteUUID.get_uuid_by_organization(self.default_org) == default_site.site_uuid
-
-    @pytest.mark.skipif(not settings.FEATURES['TAHOE_SITES_USE_ORGS_MODELS'],
-                        reason='Runs only when TAHOE_SITES_USE_ORGS_MODELS is on')
-    def test_get_organization_by_uuid_with_org(self):
-        """
-        Test get_organization_by_uuid helper when edx-organizations customization is on
-        """
-        assert TahoeSiteUUID.get_organization_by_uuid(self.default_org.edx_uuid) == self.default_org
-
-    @pytest.mark.skipif(not settings.FEATURES['TAHOE_SITES_USE_ORGS_MODELS'],
-                        reason='Runs only when TAHOE_SITES_USE_ORGS_MODELS is on')
-    def test_get_uuid_by_organization_with_org(self):
-        """
-        Test get_uuid_by_organization helper when edx-organizations customization is on
-        """
-        assert TahoeSiteUUID.get_uuid_by_organization(self.default_org) == self.default_org.edx_uuid
-
-    @staticmethod
-    @pytest.mark.skipif(settings.FEATURES['TAHOE_SITES_USE_ORGS_MODELS'],
-                        reason='Runs only when TAHOE_SITES_USE_ORGS_MODELS is off')
-    def test_create_organization_signal():
+    def test_create_organization_signal(cls):
         """
         Verify that creating an Organization object will signal a creation for TahoeUUISite
         """
         sites_count = TahoeSiteUUID.objects.count()
-        organization = Organization.objects.create(
+        organization = cls._create_organization(
             name='dummy organization',
-            description='dummy organization description',
-            active=True,
             short_name='DO',
         )
         assert TahoeSiteUUID.objects.count() == sites_count + 1
         assert TahoeSiteUUID.objects.get(organization=organization)
 
-    @staticmethod
+    @classmethod
     @pytest.mark.skipif(settings.FEATURES['TAHOE_SITES_USE_ORGS_MODELS'],
                         reason='Runs only when TAHOE_SITES_USE_ORGS_MODELS is off')
-    def test_save_organization_dose_not_create_site():
+    def test_save_organization_dose_not_create_site(cls):
         """
         Verify that saving an Organization object will not create another TahoeUUISite
         """
         sites_count = TahoeSiteUUID.objects.count()
-        organization = Organization.objects.create(
+        organization = cls._create_organization(
```

Comment:
`self` has access to all class methods without a problem.

```suggestion
        organization = self._create_organization(
```